octopus_deploy/datadog_checks/octopus_deploy/check.py

```diff
@@ -255,26 +255,33 @@ def _calculate_task_times(self, task):
             completed_time = -1
         return queued_time, executing_time, completed_time
 
-    def _process_tasks(self, space_name, project_name, tasks_json):
+    def _process_tasks(self, space_id, space_name, project_name, tasks_json):
         self.log.debug("Discovered %s tasks for project %s", len(tasks_json), project_name)
         for task in tasks_json:
             task_id = task.get("Id")
+            task_name = task.get("Name")
             tags = self._base_tags + [
                 f'space_name:{space_name}',
                 f'project_name:{project_name}',
                 f'task_id:{task_id}',
-                f'task_name:{task.get("Name")}',
+                f'task_name:{task_name}',
                 f'task_state:{task.get("State")}',
+                f'server_node:{task.get("ServerNode")}',
```

Comment:
```suggestion
                f'server_node:{task_server_node}',
```
To stay consistent with the other properties, could we replace this with a variable?